api/Navigator.json

```diff
@@ -2816,6 +2873,62 @@
             "standard_track": true,
             "deprecated": true
           }
+        },
+        "returns_plugins": {
```

Comment:
This is the "inverted" subfeature indicating when Navigator.plugins returns actual plugins. Following the version_removed it returns hard coded values used to mock the `Navigator.pdfViewerEnabled` property.